src/app/lib/config/toggles/index.js

```diff
@@ -152,7 +152,7 @@ const toggles = {
       enabled: false,
     },
     cinemaModeTV: {
-      enabled: false,
+      enabled: true,
```

Comment:
I suspect there might be a unit test which will fail as a result - PageHandlers perhaps (can't remember off the top of my head)